chia/consensus/block_body_validation.py

```diff
@@ -356,18 +362,18 @@ async def validate_block_body(
                 new_unspent: CoinRecord = CoinRecord(
```

Comment:
the name `new_unspent` is a bit misleading, especially as it's created in the `spent` state. The change looks reasonable though